<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>571</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Codec</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Raw Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtRaw"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Operation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnEncoding">
             <property name="text">
              <string>Encoding</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDecoding">
             <property name="text">
              <string>Decoding</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnExchange">
             <property name="text">
              <string>Exchange</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnClear">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cboxCodecType">
             <item>
              <property name="text">
               <string>Base64</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Result Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="txtResult"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Json</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="txtJson"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="title">
           <string>Operation</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btnJsonFormat">
             <property name="text">
              <string>Format</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnJsonCompress">
             <property name="text">
              <string>Compress</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnJsonEscape">
             <property name="text">
              <string>Escape</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnJsonDeescape">
             <property name="text">
              <string>De-Escape</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnJsonCopy">
             <property name="text">
              <string>Copy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnJsonClear">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtResult</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>txtRaw</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
